The output is incorrect because it writes an error log with arguments that contain the format character

version [v0.1.9](https://github.com/no-src/log/releases/tag/v0.1.9)

```go
package main

import (
	"errors"

	"github.com/no-src/log"
)

func main() {
	log.Error(errors.New("not found"), "hello %s", "world%s")
}
```

Output:

```text
[2023-02-20 14:28:07] [ERROR] hello worldnot found. %!s(MISSING)
```

Expect:
```text
[2023-02-20 14:28:52] [ERROR] hello world%s. not found
```

--- a/formatter/json/json.go
+++ b/formatter/json/json.go
@@ -19,7 +19,7 @@ func newJsonFormatter() formatter.Formatter {
 func (f *jsonFormatter) Serialize(c content.Content) ([]byte, error) {
 	c.Log = fmt.Sprintf(c.Log, c.Args...)
 	if c.Error != nil {
-		c.Log = fmt.Sprintf(c.Log+". %s", c.Error)
+		c.Log = c.Log + ". " + c.Error.Error()
 	}
 	buf := new(bytes.Buffer)
 	encoder := json.NewEncoder(buf)

--- a/formatter/json/json_test.go
+++ b/formatter/json/json_test.go
@@ -27,6 +27,7 @@ func TestJsonFormatter_Serialize(t *testing.T) {
 		{"debug with args and append time", content.NewContentWithTime(level.DebugLevel, nil, logTimeP, "<json formatter> %s %s", "hello", "world"), `{"level":"DEBUG","time":"2022-06-25 23:59:59","log":"<json formatter> hello world"}`},
 		{"error no args and append time", content.NewContentWithTime(level.DebugLevel, errors.New("test error"), logTimeP, "<json formatter> hello"), `{"level":"DEBUG","time":"2022-06-25 23:59:59","log":"<json formatter> hello. test error"}`},
 		{"error with args and append time", content.NewContentWithTime(level.DebugLevel, errors.New("test error"), logTimeP, "<json formatter> %s %s", "hello", "world"), `{"level":"DEBUG","time":"2022-06-25 23:59:59","log":"<json formatter> hello world. test error"}`},
+		{"error with args that contain the format character", content.NewContentWithTime(level.DebugLevel, errors.New("test error"), logTimeP, "<json formatter> %s %s", "hello%s%v", "world"), `{"level":"DEBUG","time":"2022-06-25 23:59:59","log":"<json formatter> hello%s%v world. test error"}`},
 	}
 	f := newJsonFormatter()
 	for _, tc := range testCases {

--- a/formatter/text/text.go
+++ b/formatter/text/text.go
@@ -23,7 +23,7 @@ func (f *textFormatter) Serialize(c content.Content) ([]byte, error) {
 	content := fmt.Sprintf(c.Log, c.Args...)
 	format = fmt.Sprintf(format, timeSection, c.Level.String(), content)
 	if c.Error != nil {
-		format = fmt.Sprintf(format+". %s", c.Error)
+		format = format + ". " + c.Error.Error()
 	}
 	format = formatter.AppendRowTerminator(format)
 	return []byte(format), nil

--- a/formatter/text/text_test.go
+++ b/formatter/text/text_test.go
@@ -27,6 +27,7 @@ func TestTextFormatter_Serialize(t *testing.T) {
 		{"debug with args and append time", content.NewContentWithTime(level.DebugLevel, nil, logTimeP, "<text formatter> %s %s", "hello", "world"), "[2022-06-25 23:59:59] [DEBUG] <text formatter> hello world"},
 		{"error no args and append time", content.NewContentWithTime(level.DebugLevel, errors.New("test error"), logTimeP, "<text formatter> hello"), "[2022-06-25 23:59:59] [DEBUG] <text formatter> hello. test error"},
 		{"error with args and append time", content.NewContentWithTime(level.DebugLevel, errors.New("test error"), logTimeP, "<text formatter> %s %s", "hello", "world"), "[2022-06-25 23:59:59] [DEBUG] <text formatter> hello world. test error"},
+		{"error with args that contain the format character", content.NewContentWithTime(level.DebugLevel, errors.New("test error"), logTimeP, "<text formatter> %s %s", "hello%s%v", "world"), "[2022-06-25 23:59:59] [DEBUG] <text formatter> hello%s%v world. test error"},
 	}
 	f := newTextFormatter()
 	for _, tc := range testCases {

